### ISSUE TITLE
Secondary hemolytic-uremic syndrome 

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6872]
**Preferred term label:**
Secondary hemolytic-uremic syndrome 

**Synonyms**

**Definition (free text, please give PubMed ID)**
HUS initiated by a coexisting disease or condition. The most frequently reported diseases that lead to clinically evident secondary HUS include infections, especially those caused by Streptococcus pneumoniae and the influenza virus. Secondary HUS may be associated with transplantation (solid organ or bone marrow), autoimmune disease, cancer, pregnancy and the use of certain cytotoxic drugs (e.g. calcineurin-inhibitors).
PMID: 19846853, PMID: 28416508

**Parent term (use hpo.jax.org/app)**
Hemolytic-uremic syndrome HP:0005575

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
@RichardKlaus one issue is that the HPO does not code etiologies. I think the specific kinds of HUS should be encoded as diseases in Mondo (e.g., https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0034103).
